src/main/resources/view/GoodInformation.fxml

```diff
@@ -24,6 +25,9 @@
                   <Label fx:id="goodName" styleClass="cell_big_label" text="goodName" />
                   <Label fx:id="goodQuantity" styleClass="cell_small_label" text="goodQuantity" />
                </children>
+               <GridPane.margin>
+                  <Insets bottom="5.0" left="5.0" right="5.0" top="5.0" />
+               </GridPane.margin>
```

Comment:
Why a margin?

Reply:
To prevent the text in the label being too close to the border